enterprise/server/githubapp/githubapp.go

```diff
@@ -273,6 +274,29 @@ func (a *GitHubApp) startGitHubActionsRunnerTask(ctx context.Context, event *git
 		return status.FailedPreconditionError("workflow job cannot be nil")
 	}
 
+	log.CtxInfof(ctx, "Starting ephemeral GitHub Actions runner execution for %s/%s", event.GetRepo().GetOwner().GetLogin(), event.GetRepo().GetName())
+
+	// Get an org API key for the installation. The user must have linked the
+	// installation to an org via the BuildBuddy UI, otherwise this will return
+	// NotFound.
+	installation := &tables.GitHubAppInstallation{}
+	err := a.env.GetDBHandle().NewQuery(ctx, "github_app_get_installation_for_workflow_job").Raw(`
```

Comment:
Sanity check - the installation ID is some sort of secret right? i.e. There'd be no way for people to hack into authenticating as another group with this?

Reply:
(resolved offline)